app/views/shelters/show.html.erb

```diff
@@ -4,3 +4,9 @@
 <p><%= @shelter.city %></p>
 <p><%= @shelter.state %></p>
 <p><%= @shelter.zip %></p>
+
+
+<%= link_to 'Update Shelter', "/shelters/#{@shelter.id}/edit" %><br/><br/>
+<%= link_to  "Delete Shelter", "/shelters/#{@shelter.id}", method: :delete, data: {confirm: "Confirm Sheleter Deletion"} %></br>
+
+<%= link_to 'Back to All Shelters', "/shelters" %>
```

Comment:
Spacing here is a little wonky, but is currently functional and will be addressed later on in the project.